client/styles/components/_toolbar.scss

```diff
@@ -126,6 +126,23 @@
 	}
 }
 
+.toolbar__username {
+	cursor: pointer;
+	@include themify() {
+	  color: getThemifyVariable('secondary-text-color');
+	  
+	  &:hover {
+		// Default hover color for light/dark theme
+		color: $p5-light-pink;
+  
+		// If the logo color is yellow, override the hover color
+		@if (getThemifyVariable('logo-color') == $yellow) {
```

Comment:
I think one issue here is similar to the one I noted in [your other PR](https://github.com/processing/p5.js-web-editor/pull/3397#discussion_r1999514842), where we can use just the `getThemifyVariable` utility function directly rather than the conditional.